opensrp-anc/src/main/java/org/smartregister/anc/library/viewholder/ContactTasksViewHolder.java

```diff
@@ -0,0 +1,59 @@
+package org.smartregister.anc.library.viewholder;
+
+import android.support.annotation.NonNull;
+import android.support.v7.widget.RecyclerView;
+import android.view.View;
+import android.widget.Button;
+import android.widget.ImageView;
+import android.widget.LinearLayout;
+import android.widget.RelativeLayout;
+
+import com.vijay.jsonwizard.views.CustomTextView;
+
+import org.smartregister.anc.library.R;
+import org.smartregister.anc.library.fragment.ProfileTasksFragment;
+import org.smartregister.anc.library.listener.ContactTaskDisplayClickListener;
+
+public class ContactTasksViewHolder extends RecyclerView.ViewHolder {
+    public View parent;
+    public RelativeLayout expansionPanelHeader;
+    public ImageView accordionInfoIcon;
+    public RelativeLayout expansionHeaderLayout;
+    public ImageView statusImageView;
+    public CustomTextView topBarTextView;
+    public LinearLayout contentLayout, accordionBottomNavigation;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use one line for each declaration, it enhances code readability.](https://app.codacy.com/manual/OpenSRP/opensrp-client-anc/pullRequest?prid=4809639)